src/bundle/types.ts

```diff
@@ -26,6 +26,12 @@ export type RenderPreview = (params: RenderPreviewParams) => ReactNode;
 
 export type ParseInsertedUrlAsImage = (text: string) => {imageUrl: string; title?: string} | null;
 
+export type WysiwygPlaceholderOptions = {
+    value?: string | (() => string);
+    /** Default – empty-doc */
+    behavior?: 'empty-doc' | 'empty-doc-row' | 'empty-row';
```

Comment:
Lets add something like that
https://github.com/gravity-ui/markdown-editor/blob/main/src/bundle/types.ts#L86

```
empty-doc
empty-row
empty-row-top-level
```